cmd/platform/test.go

```diff
@@ -131,7 +131,12 @@ func executeTestCommand(cmd *exec.Cmd) {
 }
 
 func runWebClientTests() {
-	os.Chdir("webapp")
+	if webappDir := os.Getenv("WEBAPP_DIR"); webappDir != "" {
```

Comment:
I don't see where we initially define or use `WEBAPP_DIR` env variable.

Reply:
It's not official defined I think (4/5), I used `WEBAPP_DIR` because was the environment variable that I used in mattermost-redux package.json `dev` and `dev:watch` tasks. We can use other environment variable name.